hibernate-core/src/main/java/org/hibernate/engine/loading/internal/LoadContexts.java

```diff
@@ -163,16 +163,16 @@ public CollectionLoadContext getCollectionLoadContext(ResultSet resultSet) {
 	 * occurs against all result-set contexts...
 	 *
 	 * @param persister The collection persister
-	 * @param ownerKey The owner key
+	 * @param key The Collection key
 	 * @return The loading collection, or null if not found.
 	 */
-	public PersistentCollection locateLoadingCollection(CollectionPersister persister, Serializable ownerKey) {
-		final LoadingCollectionEntry lce = locateLoadingCollectionEntry( new CollectionKey( persister, ownerKey ) );
+	public PersistentCollection locateLoadingCollection(CollectionPersister persister, CollectionKey key) {
```

Comment:
the javadoc of the method reads "Attempt to locate the loading collection given the owner's key. "  Is that still valid?

Reply:
Good catch, I'm going to fix it, thanks.